query05.sql

```diff
@@ -0,0 +1,43 @@
+/* query 05
+Rate neighborhoods by their bus stop accessibility for wheelchairs. Use Azavea's neighborhood dataset from OpenDataPhilly along with an 
+appropriate dataset from the Septa GTFS bus feed. Use the GTFS documentation for help. Use some creativity in the metric you devise in 
+rating neighborhoods. Describe your accessibility metric:
+
+My accessibility metric is the ratio of number of bus stops with wheelchair accessibility to neighborhood area.
+
+I couldn't get the spatial join to work, so the query comes up empty
+
+*/
+
+
+CREATE INDEX IF NOT EXISTS wheelchair_boarding__idx
+    ON septa_bus_stops
+    USING HASH (wheelchair_boarding);
+
+CREATE INDEX IF NOT EXISTS neighborhood_idx
+    ON septa_bus_stops
+    USING HASH (wheelchair_boarding);
+
+WITH neighborhood_stops AS (
+	 SELECT p.name, p.geom, b.wheelchair_boarding, p.shape_area
+     FROM neighborhoods_philadelphia AS p
+	 JOIN septa_bus_stops AS b
+	 ON ST_Within(ST_Transform(p.geom,4326), 
+				  ST_Transform(b.the_geom,4326))),
```

Comment:
It looks like your arguments to `ST_Within` are reversed. The contained should be first and the container should be second.